drawBot/context/baseContext.py

```diff
@@ -1404,6 +1404,18 @@ def listFontVariations(self, fontName=None):
             fontName = self._font
         return variation.getVariationAxesForFontName(fontName)
 
+    def listFontVariationNamedInstances(self, fontName=None):
+        """
+        List all named instances for the current font.align
```

Comment:
Align? Also, mention variable fonts here, and what it will return if the font is no variable font or has no named instance.

Reply:
typo...